core/domain/draft_upgrade_services.py

```diff
@@ -106,23 +106,18 @@ def _convert_states_v30_dict_to_v31_dict(cls, draft_change_list):
                     change.property_name ==
                     exp_domain.STATE_PROPERTY_RECORDED_VOICEOVERS):
                 # Get the language code to access the language code correctly.
-                new_changes = change.new_value['voiceovers_mapping']
+                voiceover_changes = change.new_value['voiceovers_mapping']
```

Comment:
Let's call the variable new_voiceovers_mapping.

Reply:
Variable name changed to new_voiceovers_mapping.

Reply:
Done.